service/controller/resource/prometheus/resource.go

```diff
@@ -168,6 +168,12 @@ func toPrometheus(v interface{}, config Config) (metav1.Object, error) {
 					// memory: 5Gi
 					corev1.ResourceMemory: *resource.NewQuantity(5000*1024*1024, resource.BinarySI),
 				},
+				Limits: corev1.ResourceList{
+					// cpu: 100m
+					corev1.ResourceCPU: *resource.NewMilliQuantity(100, resource.DecimalSI),
```

Comment:
This should be much higher IMO

Reply:
Like 1 CPU at least

Reply:
right, I just checked some number to be sure
![image](https://user-images.githubusercontent.com/6536819/100092388-42efa980-2e56-11eb-9fab-69d1484be45c.png)